OmniKit/PumpManager/OmnipodPumpManager.swift

```diff
@@ -1006,38 +1023,56 @@ extension OmnipodPumpManager {
         }
     }
 
-    public func readFlashLog(completion: @escaping (Error?) -> Void) {
-        // use hasSetupPod to be able to read the flash log from a faulted Pod
+    private func pulseLogString(pulseLogEntries: [UInt32], lastPulseNumber: Int) -> String {
```

Comment:
This should be in PodInfoPulseLogRecent, since it's a serialization of that object.